lib/urldata.h

```diff
@@ -201,6 +201,8 @@
 /* Download buffer size, keep it fairly big for speed reasons */
 #undef BUFSIZE
 #define BUFSIZE CURL_MAX_WRITE_SIZE
+#define MAX_BUFSIZE 524288
```

Comment:
I took 512KB as reference from https://daniel.haxx.se/blog/2014/05/14/why-sftp-is-still-slow-in-curl/
And in terms of adhoc testing that's where the performance was equivalent to sftp on my link.
But of course, I can change it to something bigger.

Should I add the max as CURL_MAX_READ_SIZE define here?
 include/curl/curl.h 
 docs/libcurl/symbols-in-versions